pokemon_app/app/controllers/moves_pokemons_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class MovesPokemonsController < ApplicationController
+  layout 'admin'
+
+  before_action :move_page
+  before_action :navigation_params
```

Comment:
I am not sure, try to set both methods in just one line, do not remember if it is possible or not, but it may worth to try.